templates/files/k8s-resource/aws-cni.yaml

```diff
@@ -153,7 +153,7 @@ spec:
       tolerations:
         - operator: Exists
       initContainers:
-        - image: {{.DockerRegistry}}/giantswarm/aws-cni-init:v1.10.1
+        - image: {{.DockerRegistry}}/giantswarm/aws-cni-init:v1.9.3
```

Comment:
so you gave up on this?

Reply:
yeah, it does not work for some reason and I just want to close the whole thing for now and maybe came back later